File: Sunrin-ICPC-2022/Pre-A/PA.py
def f(h, w):
    if h <= 140: return 6
    if h < 146: return 5
    if h < 159 or h >= 204: return 4
    bmi = w * 10000 / (h*h)
    if 159 <= h < 161:
        if 16.0 <= bmi < 35.0: return 3
        else: return 4
    if 20.0 <= bmi < 25.0: return 1
    if 18.5 <= bmi < 20.0 or 25.0 <= bmi < 30.0: return 2
    if 16.0 <= bmi < 18.5 or 30.0 <= bmi < 35.0: return 3
    return 4

T = int(input())
for tc in range(T):
    h, w = map(int, input().split())
    print(f(h, w))

File: Sunrin-ICPC-2022/Pre-A/PA.js
const fs = require('fs');
const input = fs.readFileSync('/dev/stdin').toString().split('\n');

function InRange(s, e, bmi){
    return s <= bmi && bmi < e;
}

function f(h, w){
    if(h <= 140) return 6;
    if(h < 146) return 5;
    if(h < 159 || h >= 204) return 4;
    const bmi = w / (h/100.0) / (h/100.0);
    if(159 <= h && h < 161){
        if(InRange(16.0, 35.0, bmi)) return 3;
        else return 4;
    }
    if(InRange(20.0, 25.0, bmi)) return 1;
    if(InRange(18.5, 20.0, bmi) || InRange(25.0, 30.0, bmi)) return 2;
    if(InRange(16.0, 18.5, bmi) || InRange(30.0, 35.0, bmi)) return 3;
    return 4;
}

const T = parseInt(input[0]);
for(let i=1; i<=T; i++){
    const line = input[i].split(' ');
    const h = parseInt(line[0]);
    const w = parseInt(line[1]);
    console.log(f(h, w));
}